scripts/merge-buddyup.sh

```diff
@@ -0,0 +1,22 @@
+#!/bin/bash
+
+# syntax:
+# merge-buddyup.sh locale-dir/
+
+function usage() {
+    echo "syntax:"
+    echo "merge-buddyup.sh locale-dir/"
+    exit 1
+}
+
+# check if file and dir are there
+if [[ ($# -ne 1) || (! -d "$1") ]]; then
+    usage
+fi
+
+POTFILE=`find $1 -type f -name "buddyup.pot"`
```

Comment:
Generally one would do something like 

```bash
$localeDir = $1
shift
```

To give `$1` a useful name, and also to remove it from the argument list (in case you wanted to parse other things).